ui/views.py

```diff
@@ -91,6 +91,22 @@ def get_context_data(self, video_key, **kwargs):  # pylint: disable=arguments-di
 
 
 @method_decorator(login_required, name='dispatch')
+class HelpPageView(TemplateView):
```

Comment:
Nit: we should define this under `VideoEmbed` to keep the collection/video template views defined proximally